tools/lychee/config-external.toml

```diff
@@ -19,6 +19,7 @@ exclude = [
 exclude_path = [
     "dev-docs/frozen",
```

Comment:
Probably a good idea to exclude `versioned_docs` here as well, if only to avoid getting ratelimited

Reply:
This is running once a week, the cache previously had a ttl of a day, that's why I removed it.